### PR TITLE
fix(clerk-js): Fix act output in tests [SDK-1054]

### PR DESCRIPTION
**Pull request by tmilewski**
## Description

Fixes `act` (and one `console.log`) output in Jest tests for `clerk-js`.

SDK-1054

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [x] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

File: packages/clerk-js/src/ui/components/OrganizationProfile/__tests__/InviteMembersPage.test.tsx [hunk 1]
```diff
@@ -56,14 +59,19 @@ describe('InviteMembersPage', () => {
...
+      });
       await userEvent.type(getByTestId('tag-input'), 'test+1@clerk.com,');
       await userEvent.click(getByRole('button', { name: 'Select an option' }));
       await userEvent.click(getByText('Member'));
       await userEvent.click(getByRole('button', { name: 'Send invitations' }));
-      expect(fixtures.clerk.organization?.inviteMembers).toHaveBeenCalledWith({
-        emailAddresses: ['test+1@clerk.com'],
-        role: 'basic_member' as MembershipRole,
+
+      waitFor(() => {
```
Comment (panteliselef) [inline packages/clerk-js/src/ui/components/OrganizationProfile/__tests__/InviteMembersPage.test.tsx:70]: Don't we need to add `await` ?

File: packages/clerk-js/src/ui.retheme/components/OrganizationProfile/__tests__/OrganizationMembers.test.tsx [hunk 2]
```diff
@@ -41,11 +44,11 @@ describe('OrganizationMembers', () => {
 
     fixtures.clerk.organization?.getRoles.mockRejectedValue(null);
 
-    const { getByRole } = render(<OrganizationMembers />, { wrapper });
+    const { getByRole, container } = render(<OrganizationMembers />, { wrapper });
 
-    await waitFor(() => {
-      expect(getByRole('tab', { name: 'Requests' })).toBeInTheDocument();
-    });
+    await waitForLoadingCompleted(container);
```
Comment (panteliselef) [inline packages/clerk-js/src/ui.retheme/components/OrganizationProfile/__tests__/OrganizationMembers.test.tsx:49]: 💯

File: packages/clerk-js/src/ui.retheme/components/OrganizationProfile/__tests__/OrganizationMembers.test.tsx [hunk 3]
```diff
@@ -251,15 +247,13 @@ describe('OrganizationMembers', () => {
...
-      expect(queryByText(/displaying/i)).toBeInTheDocument();
-      expect(queryByText(/1 – 5/i)).toBeInTheDocument();
-      expect(queryByText(/of/i)).toBeInTheDocument();
-      expect(queryByText(/^5/i)).toBeInTheDocument();
-      expect(getByText(/next/i)).toBeDisabled();
-    });
+    await waitForLoadingCompleted(container);
+
+    const pagination = getByText(/displaying/i).closest('p');
+    expect(pagination?.textContent).toEqual('Displaying 1 – 5 of 5');
```
Comment (panteliselef) [inline packages/clerk-js/src/ui.retheme/components/OrganizationProfile/__tests__/OrganizationMembers.test.tsx:255]: 💯

File: packages/clerk-js/src/ui.retheme/components/OrganizationSwitcher/__tests__/OrganizationSwitcher.test.tsx [hunk 4]
```diff
@@ -15,12 +15,18 @@ const { createFixtures } = bindCreateFixtures('OrganizationSwitcher');
...
   it('renders component', async () => {
+    jest.useFakeTimers();
+
     const { wrapper } = await createFixtures(f => {
       f.withOrganizations();
       f.withUser({ email_addresses: ['test@clerk.com'] });
     });
     const { queryByRole } = await act(() => render(<OrganizationSwitcher />, { wrapper }));
+
+    jest.advanceTimersByTime(15000);
```
Comment (panteliselef) [inline packages/clerk-js/src/ui.retheme/components/OrganizationSwitcher/__tests__/OrganizationSwitcher.test.tsx:26]: Do u remember what was the issue here ?

**Review by panteliselef — APPROVED** [review]
🧹  No more "act" errors 🎉

**Review by dimkl — APPROVED** [review]

**Pull request merged**